### PR TITLE
Update po_line_fund_distribution_transactions.sql

### LINKED ISSUE #626: RM/ERM - Add comments to po_line_fund_distribution_transactions
We would like to add comments to the columns. Comments from the FOLIO API documentation are preferred.

### PR DESCRIPTION
**Pull request by AlaoSUL**
Comment on Columns with descriptions from API Documentation

Closes #626 

```
All queries:
- [ ] Release notes added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

**Review by jenmawe — APPROVED** [review]
I read through the comments and saw the note in the changes.md file. As this query already existed and comments are being added, I didn't go through the entire process with the checkboxes.

File: sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql [hunk 1]
```diff
@@ -164,4 +164,68 @@ CREATE INDEX ON po_line_fund_distribution_transactions (transaction_encumbrance_
...
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.po_workflowStatus IS 'the workflow status for this purchase order';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.poline_id IS 'UUID identifying this purchase order line';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.poline_number IS 'A human readable number assigned to this PO line';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.poline_title_or_package IS 'title of the material';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.poline_listUnitPrice IS 'The per-item list price for physical or resources of \'Other\' order format';
```
Comment (amzoss) [inline sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql:179]: `\'` causes an error

File: sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql [hunk 2]
```diff
@@ -164,4 +164,68 @@ CREATE INDEX ON po_line_fund_distribution_transactions (transaction_encumbrance_
...
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.poline_quantityPhysical IS 'Quantity of physical items or resources of \'Other\' order format in this purchase order line';
```
Comment (amzoss) [inline sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql:181]: `\'` causes an error

File: sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql [hunk 3]
```diff
@@ -164,4 +164,68 @@ CREATE INDEX ON po_line_fund_distribution_transactions (transaction_encumbrance_
 
 CREATE INDEX ON po_line_fund_distribution_transactions (transaction_encumbrance_subscription);
 
+COMMENT ON COLUMN po_line_fund_distribution_transactions.po_id IS 'UUID identifying this entity';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.po_number IS 'A human readable ID assigned to this purchase order';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.po_workflowStatus IS 'the workflow status for this purchase order';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.poline_id IS 'UUID identifying this purchase order line';
```
Comment (amzoss) [inline sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql:173]: Typo in the name of the field. Should be `po_line_id`.

File: sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql [hunk 4]
```diff
@@ -164,4 +164,68 @@ CREATE INDEX ON po_line_fund_distribution_transactions (transaction_encumbrance_
...
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.fiscal_year IS 'The fiscal year';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.transaction_id IS 'UUID of this transaction';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.transaction_currency IS 'Currency code for this transaction - from the system currency';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.transaction_amount IS 'The amount of this transaction. For encumbrances: This is initialAmountEncumbered - (amountAwaitingPayment + amountExpended)';
+
+COMMENT ON COLUMN po_line_fund_distribution_transactions.transaction_encumbrance_initial_amount IS 'The initial amount of this encumbrance. Shouldn\'t change once create';
```
Comment (amzoss) [inline sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql:221]: `\'` causes an error

**Review by amzoss — CHANGES_REQUESTED** [review]
Several problems with the comments commands. Even if you're not changing the main query, it's always a good idea to run the query yourself and make sure it executes without error.

All queries:
- [x] Release notes added or updated in CHANGES.md
- [ ] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Comment by amzoss** [comment]
@jenmawe There can definitely be errors introduced, even if someone is just adding comments. I think reviewers should probably go ahead and execute the code anyway before approving.

**Review by amzoss — APPROVED** [review]
Thanks, works well now!

All queries:
- [x] Release notes added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Pull request merged**